src/test/java/edu/ucla/library/iiif/auth/delegate/CantaloupeAuthDelegateIT.java

```diff
@@ -60,6 +66,42 @@
     private static final String SINAI_ACCESS_TOKEN =
             "eyJ2ZXJzaW9uIjogIjAuMC4wLVNOQVBTSE9UIiwgInNpbmFpQWZmaWxpYXRlIjogdHJ1ZX0K";
 
+    /**
+     * A test initialization vector used to encrypt {@link #TEST_SINAI_AUTHENTICATED_3DAY}. This is just the value
+     * "0123456789ABCDEF" (see Ruby script below) encoded in hexadecimal.
+     */
+    private static final String TEST_INITIALIZATION_VECTOR = "30313233343536373839414243444546";
+
+    @SuppressWarnings("checkstyle:lineLengthChecker")
+    /**
+     * A test cookie generated using the following Ruby code, mocking the relevant part of the Sinai application:
+     * <p>
+     *
+     * <pre>
+     * #!/usr/bin/env ruby
+     *
+     * require "openssl"
+     *
+     * cipher = OpenSSL::Cipher::AES256.new :CBC
+     * cipher.encrypt
+     * cipher.key = "ThisPasswordIsReallyHardToGuess!"
+     * cipher.iv = "0123456789ABCDEF"
+     * puts (cipher.update("Authenticated #{Time.at(0).utc}") + cipher.final).unpack("H*")[0].upcase
+     * </pre>
+     *
+     * @see <a href=
+     *      "https://github.com/UCLALibrary/sinaimanuscripts/blob/44cbbd9bf508c32b742f1617205a679edf77603e/app/controllers/application_controller.rb#L98-L103">How
```

Comment:
I'm not sure that any more can be done besides hoping that the person updating the code to reflect those changes also updates the URL 🤷🏻

Reply:
Updates the URL in our Javadoc? That doesn't seem likely since it's across teams. I think ideally there would be some documentation we could reference, but I don't think there is.

Reply:
I guess we just have to keep it in sync when/if we notice it's obsolete.

Reply:
I don't see the probability of documentation being up-to-date improving by putting the info in different documentation, but maybe I'm wrong.

Reply:
If only it was possible to create a shared module that both Ursus and this project could import...

Reply:
Well, yes, just keeping our documentation up to date is a challenge too. :-)